go/engine/per_user_key_upgrade.go

```diff
@@ -99,12 +99,11 @@ func (e *PerUserKeyUpgrade) inner(ctx *Context) error {
 		return err
 	}
 
-	// Get pukring
-	err = e.G().BumpPerUserKeyring()
+	pukring, err := e.G().GetPerUserKeyring()
 	if err != nil {
 		return err
 	}
-	pukring, err := e.G().GetPerUserKeyring()
+	err = pukring.Sync(ctx.GetNetContext())
```

Comment:
Not necessarily for this PR, but it would be nice if the behavior of `GetPerUserKeyring` was "if the user hasn't been loaded in a while, go ahead and check their server version, and do a full load if anything's changed." Arguably almost anything that access the current user should do this, at some shared low level?